webtool/test_llm.py

```
@@ -433,3 +435,73 @@ def test_nicht_dekodierbare_eingabedatei_wird_zur_llmerror(tmp_path):
+
+
+# --- SSL-Kontext & Google-Provider (#385) ------------------------------------
+
+def test_ssl_kontext_nutzt_certifi(monkeypatch):
+    import ssl
+    import sys
+    aufrufe = {}
+
+    class FakeCertifi:
+        @staticmethod
+        def where():
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `where`

Add return type annotation: `str`

(ANN205)